src/components/artwork_filter/index.tsx

```diff
@@ -122,6 +127,9 @@ class ArtworkFilter extends React.Component<Props, State> {
           onLoadMore={() => this.handleLoadMore()}
           columnCount={3}
         />
+        <SpinnerContainer>
+          {this.state.loading ? <Spinner /> : ""}
```

Comment:
It would be better to return `null` instead of an empty string to keep the code more portable, because RN will complain about _any_ string that’s outside a `Text` component.